src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -121,7 +121,7 @@ public function onKernelResponse(ResponseEvent $event)
         if (self::DISABLED === $this->mode
             || !$response->headers->has('X-Debug-Token')
             || $response->isRedirection()
-            || ($response->headers->has('Content-Type') && !str_contains($response->headers->get('Content-Type'), 'html'))
+            || ($response->headers->has('Content-Type') && !str_contains($response->headers->get('Content-Type') ?? '', 'html'))
```

Comment:
```suggestion
            || !str_contains($response->headers->get('Content-Type') ?? '', 'html')
```

Reply:
just fewer operators

- with `has`
-- header absent => `has`
-- header present => `has` + `get` + coalesce + `str_contains` 
- with no `has`
-- header absent => `has` + `get` + coalesce + `str_contains` 
-- header present => `has` + `get` + coalesce + `str_contains` 

programmaticly it should be same

if not important we could remove

doing it?

Reply:
Ok it doesn't really matter we can keep it like this.